config/install/field.storage.node.layout_builder__layout.yml

```diff
@@ -0,0 +1,18 @@
+langcode: en
```

Comment:
Do we need this file?

Reply:
yeah. it will fail to install on `standard` profile because its a dependency of another config

Reply:
imo these are the conflicts we'd want with stanford_profile